ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1648,3 +1648,22 @@ FCU_RTC_RHDHS_DTM_DFSS:
   - fire_alarm
   - schedule_run_command
 
+FCU_DSP_VSC_STM_RTM:
+  description: "Hybrid variable refrigerant flow unit with supply and return air temperature monitoring, zone cooling and heating setpoints, fan speed command"
```

Comment:
I assume this device has a dx coil and evaporator valve but no data associated with it?

Reply:
Yes this is correct